packages/liveblocks-node/src/client.ts

```diff
@@ -194,6 +294,379 @@ export class Liveblocks {
     }
   }
 
+  /* -------------------------------------------------------------------------------------------------
+   * Room
+   * -----------------------------------------------------------------------------------------------*/
+
+  /**
+   * Returns a list of your rooms. The rooms are returned sorted by creation date, from newest to oldest. You can filter rooms by metadata, users accesses and groups accesses.
+   * @param params.limit (optional) A limit on the number of rooms to be returned. The limit can range between 1 and 100, and defaults to 20.
+   * @param params.startingAfter (optional) A cursor used for pagination. You get the value from the response of the previous page.
+   * @param params.userId (optional) A filter on users accesses.
+   * @param params.metadata (optional) A filter on metadata. Multiple metadata keys can be used to filter rooms.
+   * @param params.groupIds (optional) A filter on groups accesses. Multiple groups can be used.
+   * @returns A list of rooms.
+   */
+  public async getRooms(
+    params: {
+      limit?: number;
+      startingAfter?: number;
+      metadata?: RoomMetadata;
+      userId?: string;
+      groupIds?: string;
```

Comment:
We should accept an array of strings here, right?

```ts
// Currently not possible
const groupIds = ["design", "engineering"];
liveblocks.getRooms({ groupIds });
```

Reply:
The backend still decodes it as a string however. We could accept an array and then join the array to convert to string before passing it to the request. Is that what you you referring to too @CTNicholas ?

Reply:
OMG, you're right @nimeshnayaju! I wonder why this is… 🤔 

https://github.com/liveblocks/liveblocks-cloudflare/blob/2ee8995fc5646e71bf7af5a8c3a2a17412568f64/packages/liveblocks-cloudflare/src/api/v2/get-rooms.ts#L77-L86

cc @FlowFlorent @ofoucherot Any idea why this historically isn't taking in this value as an array of strings instead?

Reply:
This is because groupIds in this case is coming from a query parameter
ex: `/v2/rooms?groupIds=marketing,engineering`

We could take in an array and transform it to a query parameter string, that would be the easiest for the user I think.